app/uk/gov/hmrc/gform/formtemplate/FormComponentMaker.scala

```diff
@@ -120,7 +121,7 @@ class FormComponentMaker(json: JsValue) {
   lazy val displayWidth: Option[String] = (json \ "displayWidth").asOpt[String]
   lazy val toUpperCase: UpperCaseBoolean = (json \ "toUpperCase").asOpt[UpperCaseBoolean].getOrElse(IsNotUpperCase)
   lazy val displayInSummary: DisplayInSummary =
-    (json \ "displayInSummary").asOpt[DisplayInSummary].getOrElse(IsNotDisplayInSummary)
+    (json \ "displayInSummary").asOpt[DisplayInSummary].getOrElse(No)
```

Comment:
It would be better to import only 
```
import uk.gov.hmrc.gform.sharedmodel.formtemplate.DisplayInSummary
```

And use it as `DisplayInSummary.No`

Reply:
ok yeah i see - updated!